### DISCUSSION TITLE
Which font is needed for the unicode fonts?

### DISCUSSION THREAD

**Discussion by holgerschurig** [kickingvegas/casual-calc#170] [General]
Some of the unicode characters don't show up with the fonts I have. That is, some of the glyphs are shown, e.g. "Q" I see the roots glyph. But others are missing, e.g. after "^" I only see garbage, not the y^x glyph like on the screenshot in the README.

That one works:

```
             position: 719 of 9167 (8%), column: 10
            character: √ (displayed as √) (codepoint 8730, #o21032, #x221a)
              charset: unicode-bmp (Unicode Basic Multilingual Plane (U+0000..U+FFFF))
code point in charset: 0x221A
               script: symbol
               syntax: _ 	which means: symbol
             category: .:Base, c:Chinese, h:Korean, j:Japanese
             to input: type "C-x 8 RET 221a" or "C-x 8 RET SQUARE ROOT"
          buffer code: #xE2 #x88 #x9A
            file code: #xE2 #x88 #x9A (encoded by coding system utf-8-unix)
              display: by this font (glyph code):
    ftcrhb:-PfEd-MesloLGS NF-regular-normal-normal-*-13-*-*-*-m-0-iso10646-1 (#x724)
```

and that not:

```
             position: 816 of 9167 (9%), column: 10
            character: 𝑦 (displayed as 𝑦) (codepoint 119910, #o352146, #x1d466)
              charset: unicode (Unicode (ISO10646))
code point in charset: 0x1D466
               script: mathematical
               syntax: w 	which means: word
             category: .:Base, L:Strong L2R
             to input: type "C-x 8 RET 1d466" or "C-x 8 RET MATHEMATICAL ITALIC SMALL Y"
          buffer code: #xF0 #x9D #x91 #xA6
            file code: #xF0 #x9D #x91 #xA6 (encoded by coding system utf-8-unix)
              display: no font available
```

(above is from C-h i on the source of transient-define-prefix)

**Comment by kickingvegas** [comment]
@holgerschurig Thanks for sharing this.

BTW, are you using Windows? A similar issue has been reported in #171. As I only have access to macOS and Linux and have not seen this issue on either platform, it may be an issue with the default fonts installed on Windows.

I found this link that might perhaps help in getting/using a font that will support the math symbols used by Casual Calc. https://sites.psu.edu/symbolcodes/accents/math/

That said, this issue has popped up frequently enough that I'll make an upcoming change to optionally display Unicode symbols.

**Comment by kickingvegas** [comment]
@holgerschurig The latest release v1.11.2 now has optional support for Unicode symbols in labels. Please try that out.
